Shifty.App/Pages/IssueVoucher.razor

```diff
@@ -0,0 +1,26 @@
+@page "/Voucher"
+@using Components
+@inject NavigationManager NavManager
+
+<MudMainContent>
```

Comment:
I don't think you need the maincontent tag, do you?

Reply:
It gave some pretty styling but you are right, it was a duplicated (nested) class="mud-main-content" so I've hardcoded some margins instead though. I don't know if you prefer that?

The component is not necessarily phone-accessible, but I don't know if that is a priority? It is not my force...